chef/cookbooks/crowbar/recipes/prepare-upgrade-scripts.rb

```diff
@@ -158,11 +158,6 @@
 
 swift_storage = roles.include? "swift-storage"
 
-rabbitmq_server = search(:node, "run_list_map:rabbitmq-server").first
-# original DRBD setup is switched to cluster one in prepare_nodes_for_os_upgrade
-rabbitmq_cluster = rabbitmq_server[:rabbitmq][:ha][:enabled] &&
```

Comment:
this variable is still passed to the template belo

Reply:
True.